### DISCUSSION TITLE
10th Anniversary of Opentoonz (2016 - 2026) Schedule Updated

### DISCUSSION THREAD

**Discussion by RodneyBaker** [opentoonz/opentoonz#6357] [Announcements - Community]
We are fast approaching 10 years of OpenToonz—Blink and you'll miss the opportunity to celebrate the event. 

# OpenToonz 10th Anniversary

## Timeline of Version 1.8 (or... v2.0) Release  (Updated)

March16: Feature Freeze
Through mid-April: Bug fixes, translation files updates
April 15: Release of V1.8 RC
Early May: Release of V1.8 (or V1.8 RC2, if necessary)

What will determine whether a v2.0 is released is whether we can move Opentoonz to Qt6 and Apple Silicone.

The 10th anniversary release of OpenToonz is a milestone worth celebrating.
This celebration honors past achievement and paves the way for future growth and success.
Join us in the celebration!

**Comment by RodneyBaker** [comment]
Welcome Opentoonz users/developers!

Here's are some general if not random goals regarding Opentoonz source, application, usage and deployment we can consider as we move into the new year.

- [ ] Update @shun-iwasawa's contribution guidelines (if needed) - incorporate into main development documenation (Note 1)
- [x] Continuously improve nightly release process to ensure efficiency and reliability
- [ ] Post additional (portable) releases to resolve users immediate concerns
- [x] Time stamps in releases to better identify changes as aid to development / troubleshooting
- [ ] Allow highest degree of customization, simplification and optimization to core program
- [ ] Enhance developer ability to achieve maximum buy in from users/developers.
- [ ] Improve the path for users/developers bringing features/enhancements/fixes to source code
- [ ] Update documentation (Identify shortfalls/workarounds (leverage Opentooonz-docs repo)
- [ ] Update SDK documentation (document and bridge the gap between different SDKs)
- [ ] Establish Best Practice developmental guidelines - include code samples / references to code 
- [ ]  Include examples of code (references to code) in developmental documentation
- [ ] Streamline build process - encourage local building by user developers
- [ ] Improve development and build documentation
- [ ] Provide toonzscript documentation with extensive examples
- [ ] Integrate Github Copilot (which developers can now access freely) in development workflow
- [ ] Map Feature Requests onto roadmap document 
- [ ] Determine what implemented requests need documentation updates
- [ ] Identify requests that cannot or should not be implemented - establish how users can achieve the closest results to that request via workaround or use of other links in the production pipeline
- [ ] Improve users ability to resolve own issues at all knowledge levels

Note 1:  Important documentation/guidance exists in various places but should be incorporated into main documentaion where it can more easily be found.

**Comment by rijan72** [comment]
Is there plans for native arm/apple silicone support?

**Reply by RodneyBaker** [reply]
> Is there plans for native arm/apple silicone support?

As of today, yes, there is a plan for support of Apple Silicone.

**Comment by RodneyBaker** [comment]
> Is there plans for native arm/apple silicone support?

No one has committed to working on native arm/apple silicon support at this time.
When Opentoonz eventually moves to Qt6, that will increase the prospects of that development.
Opentoonz support for Apple Silicone (and/or Qt6) will signal a move to version 2.0
While not impossible this is very unlikely to happen through 2026.
As such the plan for support of Apple Silicon (and Qt6) is tentatively set to 2027 or later.

*This post updated as of 12 Sep 25

**Comment by jorgedelcampo** [comment]
Very excited with this announcement!! Any updates about roadmap progress?? How can we as users involved and support?

**Comment by RodneyBaker** [comment]
It is exciting to consider how far Opentoonz has come and where it can still go right?

Most updates are posted right here in github.
New (and current) Pull Requests (PR) provide a glimpse as to what developers are publicly working on.

How to get involved?
So many different ways.
One way is to provide feedback on reports and PR posted.

There technically is no official 'roadmap'.
The label is an indication of what developers have a stated interest in or that are more or less inevitable.

**Comment by jorgedelcampo** [comment]
Hi Rodney. Yes, is really exciting see how the software —and the community— has grown.

About the roadmap, I just want to know if the timeline you mentioned on this thread is progressing as planned. For example, Juny - July 2025 you mention feedback surveys and community events. Do you know if this is already happening and how we can participate?

Finally, would be great to see a new OpenToonz official version for this 10th anniversary.

Thanks for all support and effort to keep live this beautiful software. ❤️

**Comment by RodneyBaker** [comment]
> I just want to know if the timeline you mentioned on this thread is progressing as planned.

Oh, definitely not.
No one has done anything publicly for the 10th Anniversary release.
That's why I posted this 'topic'... to try to head that off.  ;)

Due to your interest I'll try to update that roadmap.

**Comment by RodneyBaker** [comment]
Previous postings have been updated to reflect the current release schedule for version 1.8.
To reiterate:

# OpenToonz 10th Anniversary

## Timeline of Version 1.8 Release 

- (Soft) Feature Freeze (End of December 2025)
- Release Candidates (March - April 2026)
- General Release (Prior to Anniversary)
- Official Anniversary (April 2026)*

**Comment by RodneyBaker** [comment]
Opentoonz Roadmap

There currently is no officially published Opentoonz roadmap.
Developers are free to champion things as they have time and interest.
Some developers have published elements of their personal roadmaps in the form of illustrations, diagrams and in the detail of postings.  Where possible it is recommended to align developmental efforts with those mappings in order to avoid redundancy or unnecessary duplication. 

Informal Roadmap
Link:  https://github.com/opentoonz/opentoonz/issues?q=is%3Aissue%20state%3Aopen%20label%3Aroadmap

Issues labeled 'roadmap' are deemed to be in the general pipeline for development although that does not suggest anyone is actively working on that task.  Some items are roadmapped simply because general development is logical or obvious.  Some items roadmapped may be years away from being incorporated into the software.  Many issues and feature requests simply have not yet been added to the roadmap.

**Comment by RodneyBaker** [comment]
Our highest priority is given to crashes.

Most reports labeled are immediately addressed where possible.
Crash reports that aren't immediately addressed will often require extensive information from the user experiencing the crash in order to fully resolve the problem.  Please consider hardware as well as software when investigating these reports.  A goal is to have the current nightly release be the most stable release available.  If other releases are even more stable and optimized that's great too.   

Reports labeled with crash:  https://github.com/opentoonz/opentoonz/issues?q=is%3Aissue%20state%3Aopen%20label%3Acrash

**Comment by Beryesa** [comment]
Hi, are there any plans to update the [website](https://github.com/opentoonz/opentoonz.github.io) as well? I'd like to contribute, but the PRs look a bit dead at the moment. 

It would be really cool to collect some promotion material for the 10th anniversary, including advertising Linux support.  
Do you know who owns the other opentoonz.org domain btw?

**Comment by RodneyBaker** [comment]
Hi @Beryesa 
Technically speaking there is no official Linux support.
There are the two automated github builds and of course everyone is free to build Opentoonz for Linux.

No, we don't have any official confirmation of who owns opentoonz.org.
As far as I know they have never bothered to contact anyone actually affiated with Opentoonz, Dwango, Digital Video, etc.

The URL http://opentoonz.com/ appears to be available but I'll guess they want a pretty price for it.

Regarding Pull Requests for the official Opentoonz website, here are the current PRs:  https://github.com/opentoonz/opentoonz.github.io/pulls

Anyone who wishes to add things should submit a PR there.

As that is the official website which isn't expected to change a lot its hard to say what changes will be entertained.  However, now is a very good time to start pressing into that if we expect to see anything related to a 10th Anniversary celebration.  I fully do expect @shun-iwasawa will update the Case Studies page as that is where most of the dated events from the Japanese anime industry using Opentoonz are captured.

**Comment by Beryesa** [comment]
Is it possible to at least word it in a way to indicate that it is _available_ cross-platform (community maintained, unofficially etc.)
Maybe in a way like the current notice for GTS
> ※ GTS is available for Windows only

or as a little addition to the open source section instead?

Edit: I didn't notice GTS was also unofficially buildable (should've checked) - exactly why I was asking about it (to let users know:D)

**Comment by RodneyBaker** [comment]
> Is it possible to at least word it in a way to indicate that it is cross-platform (community maintained etc.)

Definitely, a note could be added that states something to the effect:

"While there is no official support for Opentoonz on Linux operating systems there are a number of Linux options supported by the Opentoonz community".   A link could then be supplied to the github artifacts like:  https://github.com/opentoonz/opentoonz/actions/workflows/workflow_linux.yml

Or perhaps better yet a link could be created pointing specifically to the Opentoonz manual where Linux is mentioned:  https://opentoonz.readthedocs.io/en/latest/installing_opentoonz.html?highlight=linux#installing-on-linux-1

It is there at that location in the Opentoonz manual where other builds could be linked (Snap, etc.)

**Comment by RodneyBaker** [comment]
Happy New Year!

We have now hit a 'soft' feature freeze and development is shifted to bug fixes, optimization and stability.
Why a 'soft' feature freeze?

It is known that there are some features still planned to move forward but time hasn't allowed progress to be made.  There is still time to get features into the version 1.8 release but it is important any changes be as stable as possible.  
How does this impact your planned addition via Pull Request?

Pull Requests for features will be considered for merger but will only move forward if deemed essential and stable.
If the change will unnecessarily increase risk then merger will be delayed until after v1.8 release unless concerns can be addressed.

Not sure how best to proceed?
When in doubt consider opening a Work In Progress (WIP) PR to capture the current state of your development.
That will signal your interest in seeing that development move forward.
While there is no guarantee the change will be added to the v1.8 release users will be able to download, test and use the CI builds with that change.

Feedback on how we are moving forward toward this important 10th anniversary release is appreciated.
Thanks!

**Comment by RodneyBaker** [comment]
Please see:  https://github.com/opentoonz/opentoonz/discussions/6421 where we'll share some artwork and graphics related to the 10th Anniversary of Opentoonz.

**Comment by shun-iwasawa** [comment]
Dear OpenToonz community,

Thank you for your continued contributions to OpenToonz.
I sincerely apologize for the significant delay in releasing the next version of OpenToonz beyond the original schedule due to my own laziness. Although delayed from the original plan, I'd like to proceed with the following schedule:

March16: Feature Freeze
Through mid-April: Bug fixes, translation files updates
April 15: Release of V1.8 RC
Early May: Release of V1.8 (or V1.8 RC2, if necessary)

If you have any thoughts regarding the above schedule, please let me know.
(I'll stick this schedule on my desk and will remind myself every day.)

Furthermore, the next version will (hopefully) be V2.0, and I intend to focus primarily on migrating to Qt 6.x and developing an Apple Silicon version.

**Reply by gab3d** [reply]
That last sentence is especially encouraging for the future of the program. Thanks!! 💪

**Reply by ErwinBr** [reply]
Thank you very much for the update. The Apple Silicon version will be much anticipated. I'm happy to hear it being mentioned, as I've heard that Apple will drop Rosetta (and therefore non-Silicon support) on macOS 28, which could be released in 2027 already!

**Reply by shun-iwasawa** [reply]
Dear OpenToonz community, I apologize for the delay in releasing V1.8RC.

Actually, I am still working on a scene for the feature film currently in production in our studio.
The scene I’m currently working on (using IwaWarper) is the final one: it’s both the last part of my assigned work and is the very last scene of the film.

My work has taken longer than expected, and for that reason, I’m very sorry to say that I must push back the release schedule for V1.8RC (and V1.8) by a few weeks.
I sincerely appreciate your understanding and patience.

**Reply by jorgedelcampo** [reply]
Thanks Shun. It is totally understandable and, at the same time, exciting because it shows that the project is still alive!

**Reply by Abel032** [reply]
Looking forward to the film and OpenToonz 1.8RC (⁠ ⁠╹⁠▽⁠╹⁠ ⁠)

**Reply by age3rcm** [reply]
F5555555555555

**Reply by shun-iwasawa** [reply]
Sorry to keep you waiting—I finally finished the scene I was working on today. I’ll be working on fixing bugs in OpenToonz and preparing for the next version release.

**Reply by pppoofin** [reply]
Very nice to hear, do you think the development of infrastructure tools like claude code makes this qt6 port more of a manageable task? (Im not a bot lol) Ive been waiting for a silicon port for a while, and I remember the main barrier to the apps update was the time and complexity needed. Also if needed I would be interested in a community kick-starter if that would help you guys in any way as well.

**Reply by ErwinBr** [reply]
<img width="701" height="375" alt="image" src="https://github.com/user-attachments/assets/9c1df83b-1bcc-4813-b0b0-d1af83f62a91" />

I haven't seen this warning before, but it confirms that the upgrade to Apple Silicon is becoming rather urgent. Do we know how long Rosetta will be supported on Mac?

**Reply by shun-iwasawa** [reply]
I just uploaded the V1.8RC installers to the [Releases](https://github.com/opentoonz/opentoonz/releases/tag/v1.8.0rc) section.
Could someone please verify that the installers works properly before we update the OpenToonz website to announce it to the public 🙏 ?

**Reply by jorgedelcampo** [reply]
Hi Shun. Installer tested successfully on **Windows 11 Pro**.
<img width="772" height="288" alt="image" src="https://github.com/user-attachments/assets/f615eab0-64ee-4c59-8bbb-dd9abf4d5915" />

**Reply by RodneyBaker** [reply]
Installer for v1.8RC is working here! :)

**Reply by age3rcm** [reply]
it opens!
unfortunately, intel GPU still has that weird delay on complex projects on windows, except on linux

**Comment by konero** [comment]
Might I suggest snapshotting 1.8 to a commit before the Tool Properties panel/Pinning feature was introduced? I feel it was included too early. There's a lot of UX, clipping, pinning, unconventional styles and layout issues we've run into, and as it stands the feature is incomplete, much of it is beyond bug fixing, to allow discussion to potentially improve some of these problems before it goes mainstream. Some issues I cannot reproduce reliably, for example shrinking the width of the window caused a hard-crash (no crash popup). I wish I had more time to test things, but we're running into a vast swathe of issues at the moment.

I like the idea of Qt modernization, but perhaps also these large changes could be deferred to 1.9/2.0? To allow further beta testing.

**Reply by shun-iwasawa** [reply]
I fully understand your concerns. It’s true that these panels still seem to be lacking in functionality.
On the other hand, I think there’s also an advantage to including features under development in the release version, as it allows us to gather feedback from more active community members.

Therefore, as a middle ground, I plan to add the labels `[WIP]` or `[Experimental]` to the Tool Properties and Brush Preset panels in V1.8 to indicate that these features are still under development.

As for the crash you encountered, I haven’t been able to reproduce it on my environment yet. I’d like to fix it during the V1.8 RC period if possible, but I’d need a bit more information.

I appreciate your understanding.

**Comment by jorgedelcampo** [comment]
Hi folks!
Besides the upcoming v1.8 OpenToonz release and due technical discussion, what about the 10th anniversary activities? We will do anything special?

**Comment by RodneyBaker** [comment]
As of this moment nothing special is planned.
It's not too late!

**Comment by RodneyBaker** [comment]
Opentoonz v1.8 has been released.

https://github.com/opentoonz/opentoonz/releases/tag/v1.8.0

Go get it!

Thanks to all who have helped us get to this 10th year anniversary!
